### PR TITLE
README update

### PR DESCRIPTION
**Pull request by alisterburt**
this PR is a WIP for improvement of the README (#9)

I started by adding just a couple of lines in an overview section.

Questions:
- should we add examples to the readme at this stage or just point to the examples folder?
- what's your opinion on adding an image to the README? I've heard conflicting viewpoints 
- any pointers for other things you think are missing?

### CODE REVIEW

**Comment by sofroniewn** [comment]
> should we add examples to the readme at this stage or just point to the examples folder?

Might be nice to have some minimal code snippet that shows how to use the animation functionality headless, and then link to the examples

> what's your opinion on adding an image to the README? I've heard conflicting viewpoints

We can have one nice image of the viewer with the animation widget - can contain your data from https://twitter.com/AlisterBurt/status/1356631458941698048

We're trying to get to automated screenshots/ movies within the napari docs and will probably want to use this tool there, but having once nice screenshot image as a png in the readme is a good idea I think

File: README.md [hunk 1]
```diff
@@ -6,23 +6,69 @@
...
+## Examples
+Examples can be found in our [examples](examples) folder. Simple examples for both interactive and headless 
+use of the plugin follow.
+
+### Interactive
+**napari-animation** can be used interactively by creating an `AnimationWidget` from a napari `Viewer` and adding it to
+the viewer as a dock widget.
+
+```python
+animation_widget = AnimationWidget(viewer)
```
Comment (sofroniewn) [inline README.md:42]: Can you add the `AnimationWidget` import here too

File: README.md [hunk 2]
```diff
@@ -6,23 +6,69 @@
...
+![AnimationWidget image](resources/screenshot-animation-widget.png)
+
+### Headless
+**napari-animation** can also be run headless, allowing for reproducible, scripted creation of animations.
+
+```python
+from napari_animation import Animation
+
+animation = Animation(viewer)
+viewer.update_console({'animation': animation})
```
Comment (sofroniewn) [inline README.md:55]:
```suggestion
```
We can drop that console update line. We should probably remove from examples scripts too. It's not necessary

**Review by sofroniewn — APPROVED** [review]
Love this! Great screenshot!

I left two very minor comments I'm good merging this then. We can continue to improve and edit as we get closer to release, but these changes are already nice to have public.

**Comment by sofroniewn** [comment]
Thanks for the fixes! Merging now!!

**Comment by alisterburt** [comment]
:tada: :tada: :tada:

**Pull request merged**